.github/workflows/github.yml

```diff
@@ -244,7 +244,7 @@ jobs:
 
     - name: CMake Preset with ccache
       run: |
-        cmake -DCMAKE_CXX_COMPILER_LAUNCHER=ccache --preset ${{ matrix.preset }}
+        cmake -DENABLE_CCACHE:BOOL=ON --preset ${{ matrix.preset }}
```

Comment:
I didn't know the `:BOOL` syntax, is there a benefit to it?

Reply:
I think just using "ON" would work too, but when the type is annotated CMake will barf if user makes a typo when typing ON or OFF.